schema/schema.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE item (
+    id  INT GENERATED ALWAYS AS IDENTITY,
+    genre_id INT,
+    author_id INT,
+    label_id INT,
+    publish_date DATE,
+    archived BOOLEAN,
+    PRIMARY KEY(id),
+    FOREIGN KEY (genre_id) REFERENCES genres (id),
+    FOREIGN KEY (author_id) REFERENCES authors (id),
+    FOREIGN KEY (label_id) REFERENCES labels (id)
+);
+
+CREATE TABLE music_albums (
+    id  INT,
+    name VARCHAR(100),
+    on_spotify BOOLEAN,
+    PRIMARY KEY(id),
+    FOREIGN KEY(id) REFERENCES item(id)
+);
+
+CREATE TABLE genres (
```

Comment:
As `music_albums` (`item`) has a `FOREIGN KEY` referencing to this table, you should create this table first. Please, move the `genres` table definition to the beginning of the script.